kustomize-lint: init at 1.0.0

matanyall opened the pull request:
A linter for Kustomize configuration files that detects unreferenced files and validates that all referenced files exist.

Homepage: https://github.com/groq/kustomize-lint

## Things done

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [x] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR.
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

iamanaws commented on pkgs/by-name/ku/kustomize-lint/package.nix:
```diff
@@ -0,0 +1,42 @@
+  src = fetchFromGitHub {
+    owner = "groq";
+    repo = "kustomize-lint";
+    rev = "v${version}";
```
`tag` is preferred [Nixpkgs Reference](https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub)

```suggestion
    tag = "v${version}";
```

iamanaws reviewed (changes requested):
Welcome!

- Please fix the small change below
- And then squash the last commits into `kustomize-lint: init at 1.0.0`

So we end only with

- `maintainers: add matanyall`
- `kustomize-lint: init at 1.0.0`

matanyall committed:
maintainers: add matanyall

Amp-Thread-ID: https://ampcode.com/threads/T-019b6af9-4824-7638-8c1b-81c26fec93b8
Co-authored-by: Amp <amp@ampcode.com>

matanyall committed:
kustomize-lint: init at 1.0.0

iamanaws reviewed (approved):
thanks!

matanyall commented:
Hey @iamanaws, I'm new to nixpkg land, but is there something I'm missing that's making this take so long? Anything I can do to help it goes though?

iamanaws commented:
Hey, everything is right, but need to be patient until someone with commit bit give it a look and got this merged. We are short on committers, so it could take a few days or a few weeks. 

> [How can I get a committer to look at my PR?](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-can-i-get-a-committer-to-look-at-my-pr)
